### PR TITLE
docs: update to use @baransu/graphql_ppx_re

### PR DESCRIPTION
**Pull request by baransu**
Updating documentation to use https://github.com/reasonml-community/graphql_ppx which is now community-supported and maintained graphql_ppx for the bs-platform 6+

**Pull Request Labels**

- [ ] feature
- [ ] blocking
- [x] docs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
   ],
   "author": "Grégoire Van der Auwermeulen <gregoirevandera@gmail.com>",
   "devDependencies": {
-    "bs-platform": "^7.0.1",
+    "@baransu/graphql_ppx_re": "^0.7.1",
+    "bs-platform": "^7.2.2",
     "graphql_ppx": "^0.2.8",
```
Comment (Gregoirevda) [inline package.json:22]: Is there a dependency on graphql_ppx?

Comment (baransu) [reply package.json:22]: Ou, it should be removed as it's replaced by graphql_ppx_re

Comment (Gregoirevda) [reply package.json:22]: Can we delete?

Comment (baransu) [reply package.json]: Pushed fix ;)

**Comment by Gregoirevda** [comment]
Thanks @baransu ! Fixes: #239

**Comment by Gregoirevda** [comment]
I've just deployed under version `0.20.0`

**Pull request merged**